tools/ClassGen/InstrBuilder.h

```diff
@@ -33,6 +33,7 @@ enum class OperandKind : unsigned char {
   In,
   Out,
   InOut,
+  Scratch,
```

Comment:
Could you add a comment about Scratch operand kind? In particular, it is almost the same as "Out" in many cases. But with some differences that should be explained.

Reply:
Added a comment. Have a look.